src/web_ui/src/components/Layout/Header/HeaderBar.js

```diff
@@ -195,10 +206,30 @@ export default function HeaderAppBar() {
       open={isMenuOpen}
       onClose={handleMenuClose}
     >
-      <MenuItem onClick={handleMenuClose}>Sign In</MenuItem>
-      <MenuItem>
+      
+      {JSON.parse(localStorage.getItem("farmRaiAuth")) === null
```

Comment:
This should be in the app state, move this function call to Login action and access value from app state. 
id=**1**